WebARonARKit/WebARonARKit.js

```diff
@@ -968,6 +972,8 @@
     }
 
     callRafCallbacks();
+    
+    window.webkit.messageHandlers.WebARonARKit.postMessage("arDataWasUsed:");
```

Comment:
can we call this advanceFrame? I think it's clearer and less weird

Reply:
+1